flink-jepsen/src/jepsen/flink/db.clj

```diff
@@ -293,7 +293,6 @@
        "-Djobmanager.rpc.port=6123 "
        "-Dmesos.resourcemanager.tasks.mem=2048 "
        "-Dtaskmanager.heap.mb=2048 "
-       "-Dtaskmanager.numberOfTaskSlots=2 "
```

Comment:
Did you mean to use `taskmanager-slots` here?

Reply:
I think Mesos will pick up the configuration settings from the Flink configuration. Therefore, there should be no need to explicitly override it here. The same may apply to the other configuration settings as well. But I would like to address this in a separate issue.